src/param.js

```diff
@@ -418,3 +418,25 @@ class NumberParam extends Json {
   }
 }
 module.exports.Number = (...args) => new NumberParam(...args);
+
+class Custom extends Param {
```

Comment:
@simlu `Custom` is just a placeholder, I'm wondering if you already had a name in mind for this type of param?

Reply:
Maybe `Async`?

Reply:
Or should this just be a "remap" parameter (optional) in Param?